plugins/newsdownloader.koplugin/main.lua

```diff
@@ -107,16 +107,16 @@ function NewsDownloader:addToMainMenu(menu_items)
                 callback = function() self:removeNewsButKeepFeedConfig() end,
             },
             {
-                text = _("Force skip images download toggle"),
+                text = _("Toogle: force skip images download"),
```

Comment:
Typo, but I'm not entirely sure what the text means. I suspect that "toggle" shouldn't be there since it'd already be visually in the checkbox.
```suggestion
                text = _("Toggle: force skip images download"),
```